### ISSUE TITLE
ChocoSniff CPU usage seems to rise based on internet traffic & stuck in a usage loop.

### ISSUE THREAD

**Issue by Meowchestra** [MogAmp/MogAmp#23]
_This issue is very weird._

I noticed ChocoSniff cpu usage is entirely dependent on network saturation when you have multiple adapters. Normally on no usage, ChocoSniff uses 0.0-0.1% of the CPU _per app_ running. When I put some network usage to the test with a download, I noticed ChocoSniff **rise** in cpu usage _per app_ running based on the speed of the download.

Normal no activity / download.

![link4](https://i.imgur.com/xRECihd.png)

With a 10mb/s download while connected to a vpn;

![link](https://i.imgur.com/TYXl0qr.png)

With a 30mb/s download while connected to a vpn;

![link2](https://i.imgur.com/fcLOCkq.png)

With a 40mb/s download while connected to a vpn;

![link3](https://i.imgur.com/JPc4i4y.png)

I disconnected from my vpn and ran a google speed test (~920mbps). Somehow ChocoSniff bugged and became locked in a permanent 4% usage _per app_ no matter what, even after the speedtest stopped. It didn't drop so I just restarted MogDoot.
![link5](https://i.imgur.com/qMZgeAG.png)

Testing upload speed did not affect cpu usage, only download speed. CPU usage also only rose when you are in-game, not in the main menu of Final Fantasy. I tested in the middle of nowhere in Western Thanalan with no one else nearby - only 2 clients opened.

**Here is where things get doubly weird.**

After I disconnected from my VPN (2nd network adapter) and logged back into FFXIV (just clicking okay to lost connection to server and connecting again -- not relaunching clients nor restarting Doot), Amp/Doot showed some weird network usage feedback loop in task manager

https://drive.google.com/file/d/1PTFV1GINQ172hgBOKHeUpPxq9R3KkY9b/view

Restarting both Amp/Doot doesn't fix the issue. The network usage loop always happens when on a single adapter. Even weirder, if I start a large file download to test, Amp/Doot matches the network usage from firefox for the file downloading. It doesn't show a cyclical network usage when I am connected to my VPN (2nd adapter).

![link10](https://i.imgur.com/H4m4sRr.png)
![link11](https://i.imgur.com/TJWj9dO.png)

---

To summarize;

* Using a 2nd adapter (VPN), no network usage loop / mirroring from other apps but CPU rises based on download speed being used.
* Using a singular adapter (LAN/Ethernet - no VPN connected), network usage loops / mirrors other applications, but CPU does not rise based on network usage.

Why does this matter? It's weird... It also becomes a problem when you have 8+ ChocoSniffs running for full party Ensemble or worse... 20+ accounts open. ;-;
